charts/generic/container-kill/container-kill.chartserviceversion.yaml

```diff
@@ -2,16 +2,18 @@ apiVersion: litmuchaos.io/v1alpha1
 kind: ChartServiceVersion
 metadata:
   name: container-kill
-  version: 0.1.2
+  version: 0.1.1
```

Comment:
Leave the version as it is. We  should not be downgrading a version.

Reply:
fixed this